embedded-cal-nrf54l15/memory.x

```diff
@@ -0,0 +1,5 @@
+MEMORY
+{
+  FLASH : ORIGIN = 0x00000000, LENGTH = 1524K
+  RAM : ORIGIN = 0x20000000, LENGTH = 128K
```

Comment:
At https://www.nordicsemi.com/Products/nRF54L15, I see 1.5 MB flash, but there seems to be 256 kB of RAM. Why have 128 kB as length here?